listenbrainz/webserver/static/js/src/stats/HeatMap.tsx

```diff
@@ -36,7 +36,7 @@ export default function Heatmap(props: HeatMapProps) {
       width={width}
       height={height}
       enableLabels={false}
-      padding={1}
+      padding={4}
```

Comment:
IMO the graph looks better with padding set to 1. It doesn't look good on mobile view and the ticks look misaligned.

Reply:
okk